csrc/python_frontend/python_bindings.cpp

```diff
@@ -488,6 +488,80 @@ void initNvFuserPythonBindings(PyObject* module) {
         return ss.str();
       });
 
+  //! KernelProfiles are encapsulated in FusionProfiles where each KP
+  //! is associated with a segment.
+  py::class_<KernelProfile> kernel_prof(nvfuser, "KernelProfile");
+  kernel_prof.def_property_readonly(
```

Comment:
Is there a code comment/ wiki somewhere which lists the descriptions of these field? While some are self-explanatory, others like `stream`, `correlation_id` are not straightforward.